wagtail_localize/machine_translators/deepl.py

```diff
@@ -17,8 +27,9 @@ class DeepLTranslator(BaseMachineTranslator):
     display_name = "DeepL"
 
     def translate(self, source_locale, target_locale, strings):
+        api_endpoint = self.options.get("API_ENDPOINT", "https://api.deepl.com/v2/translate")
```

Comment:
nitpick: missed the first time around, but we do prefix our settings with `WAGTAILLOCALIZE_` so this should be consistent. i.e. `WAGTAILLOCALIZE_DEEPL_API_ENDPOINT`